sycl/test-e2e/README.md

```diff
@@ -221,19 +226,32 @@ unavailable.
  * **ocloc**, **opencl-aot** - Specific AOT tool availability;
  * **level_zero_dev_kit** - Level_Zero headers and libraries availability;
  * **cuda_dev_kit** - CUDA SDK headers and libraries availability;
+ * **dump_ir**: - compiler can / cannot dump IR;
+ * **llvm-spirv** - llvm-spirv tool availability;
+ * **llvm-link** - llvm-link tool availability;
+ * **fusion**: - Runtime supports kernel fusion;
+ * **aspect-\<name\>**: - SYCL aspects supported by a device;
+ * **architecture-\<name\>** - [SYCL architecture](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc) of a device (e.g. architecture-intel_gpu_pvc);
+
+### Manually-set features
+
+The following features are only set if you pass an argument to `llvm-lit` (see
+section below). All these features are related to HW detection and they should
+be considered deprecated, because we have HW auto-detection functionality in
+place. No new tests should use those features:
```

Comment:
```suggestion
place. No new tests should use these features:
```

Reply:
Applied in 0f6733097bfb07656eedf16c751814217fb67c0a